bskyembed/.eslintrc

```diff
@@ -15,6 +15,6 @@
   "parserOptions": {
     "sourceType": "module",
     "ecmaVersion": "latest",
-    "project": "./tsconfig.json"
+    "project": "./bskyembed/tsconfig.json"
```

Comment:
?

Reply:
Eslint wouldn't let me merge unless I did this - for some reason it looks at this path from the perspective of the repo root? Very odd, I don't know if there's something else funny going on